pkg/systemd/terminal.jsx

```diff
@@ -51,6 +54,11 @@ const _ = cockpit.gettext;
             this.setState({ title: title });
         }
 
+        onThemeChanged(value) {
+            this.setState({ theme: value });
+            cockpit.localStorage.setItem('terminal-theme', value);
```

Comment:
This doesn't seem  to work for me in Firefox. I switched to light, logged out, logged back in, and ended up with dark again.